### ISSUE TITLE
Tests sometime fails when creating topics

### ISSUE THREAD

**Issue by morenol** [infinyon/fluvio-client-python#251]
```
subprocess.CalledProcessError: Command 'fluvio topic create 90c7826f-b63a-4e72-b382-a5e2a581bb4d' returned non-zero exit status 1.
```

**Comment by EstebanBorai** [comment]
Interesting. No other logs are provided?

**Comment by morenol** [comment]
We would need to collect SC and SPU logs to gather more information. When I added this line the CI to behave in a better way. https://github.com/infinyon/fluvio-client-python/blob/a75e531889a1cf521d1f1b84d5e1cb9ab7800f4d/integration-tests/test_fluvio_python.py#L49

**Comment by EstebanBorai** [comment]
Mmm sounds like a race condition 🤔, I imagine Fluvio wasn't ready for the moment the `fluvio topic create` was executed?

**Comment by morenol** [comment]
I don't think so, I have replicated that issue locally. We have to dig into the actual logs of the SC/SPU

**Comment by ozgrakkurt** [comment]
> I don't think so, I have replicated that issue locally. We have to dig into the actual logs of the SC/SPU

Can you give instructions. to replicate? Do you just run `make integration-tests` repeatedly?

**Comment by morenol** [comment]
yes! and use a local fluvio instance
